storage/storage_src.go

```diff
@@ -42,8 +42,14 @@ type storageImageSource struct {
 	getBlobMutex    sync.Mutex              // Mutex to sync state for parallel GetBlob executions
 	SignatureSizes  []int                   `json:"signature-sizes,omitempty"`  // List of sizes of each signature slice
 	SignaturesSizes map[digest.Digest][]int `json:"signatures-sizes,omitempty"` // List of sizes of each signature slice
+
+	// fromDigestToTOC converts from a layer diffID returned by LayerInfosForCopy to the TOC digest.  The TOC digest
+	// must be used for lookups in the underyling storage.
+	fromDigestToTOC map[digest.Digest]digest.Digest
 }
 
+const expectedLayerDiffIDFlag = "expected-layer-diffid"
```

Comment:
What is setting this? If it is c/storage, presumably it should define a public constant.

(And, something somewhere should _precisely_ document the semantics. Is it “this is the digest that we expect, and if it doesn’t match, something is badly broken, but we haven’t (yet?) validated this DiffID, so this value must not be used for lookups”?)

Reply:
we are setting it in `storage_dest.go`: https://github.com/containers/image/pull/1980/files/cbea9be44d374151757269f4fec431acad059206#diff-2d4a826a6986cd3a8f73b9fb0e68b27e87caaefb0703a5b1695255e4d22a7d5aR673-R678

Reply:
I’m sorry, I should have found that.

Reply:
This was documented in #2288.